File: web-development/level-1/helsinki-part-1-2/Java/part1/week6/ex94/Person.java
public class Person {

  private String name;
  private String phoneNum;

  //constructors
  public Person(String name, String phoneNum) {
    this.name = name;
    this.phoneNum = phoneNum;
  }

  //behaviors methods
  public String getName() {
    return this.name;
  }

  public String getNumber() {
    return this.phoneNum;
  }

  public String toString()  {
    return this.name + ", phone no: "+ this.phoneNum;
  }

  public String changeNum(String newNumber) {
    return this.phoneNum = newNumber;
  }






//class ends here
}

File: helsinki-moocs/python/week06/ex94/Person.py
class Person:

    #consturctor
    def __init__(self, name, number):
        self.person_name = name
        self.person_number = number

    #behaviors

    def get_name(self):
        return self.person_name

    def get_number(self):
        return self.person_number

    def change_number(self, new_number):
        self.person_number = new_number
